### ISSUE TITLE
"-XX:+EnsureHashed:java/lang/Class,java/lang/Thread" added to the list of default options

### ISSUE THREAD

**Issue by pshipton** [eclipse-openj9/openj9-docs#989]
**Issue or pull request number:**
https://github.com/eclipse-openj9/openj9/pull/15444

**Overview:**
"-XX:+EnsureHashed:java/lang/Class,java/lang/Thread" added to the list of default options (options.default) to pre-hash those classes for performance. The EnsureHashed option is already documented, https://www.eclipse.org/openj9/docs/xxensurehashed/

**Release target:**
0.35 (and 22_03u1)

**Applies to the following JDK versions:**
All

**Applies to the following platforms:**
All

**Comment by Sreekala-Gopakumar** [comment]
Should I make any changes to the xoptionsfile then? Thanks!

**Comment by pshipton** [comment]
Correct, the options.default files hasn't been empty for quite a long time, pre dating OpenJ9. It's contained -Xlockword options, and now the EnsureHashed option. IBM Java contains more options as well.
